CHANGELOG.md

```diff
@@ -47,7 +67,11 @@ Changelog for NeoFS Node
 - Update `tzhash` to `v1.7.1`
 - `github.com/panjf2000/ants/v2` to `v2.8.2`
 - `github.com/nspcc-dev/neofs-contract` to `v0.18.0` (#2580)
+- `golang.org/x/net` to 0.17.0 (#2621)
```

Comment:
do we need it? i mean every `go.mod` update. or only the ones with a separate PRs that fix smth?

Reply:
I don't think they're very useful, but this particular one fixes some CVE.